### ISSUE TITLE
Loading NIFTI files with nibabel sometimes breaks them

### ISSUE THREAD

**Issue by johnmarktaylor91** [nipy/nibabel#772]
Hello, sometimes when I load a NIFTI file using nib.Nifti1Image.from_filename or nib.load, and then try to load that file again later, I get the error: "Not a gzipped file". My only interaction with these files is through nibabel, so I'm wondering I'm accidentally corrupting these files somehow? Has anyone ever run into this, and does anyone know how to prevent it?

**Comment by matthew-brett** [comment]
I don't think this is Nibabel - we only read the file, we do not write to it, unless you do that explicitly.  Could something be going wrong with your filesystem?

**Comment by johnmarktaylor91** [comment]
Hmm... to be clear what's happening, I'm loading a .nii.gz file (the beta values from a GLM) using the following snippet:

# Load runs

func_runs= []
        for r in range(nr):
            print("Run %s" % run_list_3digits[s][r])
            run_type = run_types[s][r]
            run_num = run_list_3digits[s][r]
            beta_filename = opj(bolddir,'glm_'+run_type+'_perrun_' +mc+'_'+sm,'pr'+run_num,'beta.nii.gz')
            func_runs.append(nib.load(beta_filename))

# After defining a region-of-interest mask, roi_masker:

all_betas = [roi_masker.fit_transform(func_runs[r]) for r in range(len(func_runs))]

I do this a few different times over the course of data analysis (that is, each NIFTI file ends up getting read a few different times, as it's used for different purposes). My script involves no other interaction with the NIFTI files besides reading it in with nibabel. At some point, one of the NIFTI files ends up getting "corrupted", such that nibabel throws an error when I try to read it in. I've verified that it is not corrupted in this way prior to running the script. I've tried regenerating the NIFTI files (each time verifying that they're initially alright), and this has happened every time. It doesn't happen to every file, just one. Could there be some kind of problem with the NIFTI file remaining in an "open" status, and several processes trying to open it at the same time?

**Comment by johnmarktaylor91** [comment]
Never mind--this comes from a coding error on my part.

**Comment by matthew-brett** [comment]
No problem - thanks for closing.
